oll_sc/cli.py

```diff
@@ -34,30 +56,47 @@ def check_pin(pin):
 
 @oll_sc.command()
 @click.argument('input_data')
+@click.argument('key_id', type=int)
 @click.argument('pin')
-@click.option('--key_id')
 @click.option('--output-path', '-o', type=click.Path(), default=None,
               help='The output file path to write signature to.')
-def sign_rsa_pkcs_pss_sha256(input_data, pin, key_id=None, output_path=None):
+def sign_rsa_pkcs_pss_sha256(input_data, key_id, pin, output_path=None):
```

Comment:
You've already set output_path default value.
If you prefer to use arguments here, which makes sense but does not give us the option to add help, could you add some docstrings here?

Reply:
I can add a _docstring_, but it's pretty much self-explanatory. I will fix default value, good catch.

Reply:
It's not clear that `input_data` can be either text or a file. That's the only argument that would benefit from specifying `help` or a docstring.